packages/models-library/src/models_library/services.py

```diff
@@ -58,6 +61,9 @@
 ServiceKey = constr(regex=KEY_RE)
 ServiceVersion = constr(regex=VERSION_RE)
 
+DockerImageKey = constr(regex=DOCKER_IMAGE_KEY_RE)
```

Comment:
Ok, so actually I was confused why these constants ended up in services.py.
It's because these are not service constants but more generic docker constants. What about creating a separate module? Especially since it is confusing to see these in here.

Reply:
let me move them to a separate module.